recipe/conda_build_config.yaml

```diff
@@ -1,37 +1,45 @@
 vcver:
+ - 14.2
  - 14.2
  - 14.2
  - 14.1
 vsyear:
+ - 2019
  - 2019
  - 2019
  - 2017
 vsver:
+ - 16
  - 16
  - 16
  - 15
 runtime_version:
+ - 14.29.30139
  - 14.29.30037
```

Comment:
We don't need to support two versions with the same minor version. Can you remove the `14.29.30037` version?

Reply:
Sure. But in turn, I don't understand why we even still have 14.28, which is also 16.{x=8}; 16.10 & 16.11 just happen to [share](https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering) the same runtime (minor) version.

Reply:
14.28 and 14.29 have different minor versions and we need both for users. See https://github.com/conda-forge/vc-feedstock/issues/36